### ISSUE TITLE
Debian Linux, Chrome 40.0.2214.111 (64-bit) Works fine

### ISSUE THREAD

**Issue by clemare** [jlblatt/AwSnap#3]
As I understand, the browser should crash when open this url: http://www.reddit.com/r/webdev/comments/31kumu/this_post_crashes_chrome/

However In Debian Linux 8 Jessie Chrome 40.0.2214.111 64 bits it works fine.
![bug chrome](https://cloud.githubusercontent.com/assets/8670788/7051480/1e823f8e-ddff-11e4-8708-2a04e03aa71f.png)

**Comment by jlblatt** [comment]
Yes, it appears the issue was with v41 only and has since been fixed
